stdlib/3/types.pyi

```diff
@@ -3,9 +3,14 @@
 
 # TODO parts of this should be conditional on version
 
-from typing import Any, Callable, Dict, Iterator, Optional, Tuple, TypeVar, Union, overload
+from typing import (
+    Any, Callable, Dict, Generic, Iterator, Mapping, Optional, Tuple, TypeVar,
+    Union, overload
```

Comment:
I've added `Mapping` and `Generic` to this already long list. I see that `Union` is not used after my changes anymore. I'll remove it.